packages/code-studio/src/styleguide/SpectrumComparison.tsx

```diff
@@ -44,6 +44,10 @@ const options = [
   { title: 'Three', value: '3' },
 ];
 
+function noop(): void {
+  // no-op
+}
+
```

Comment:
You can use the `EMPTY_FUNCTION` constant instead from `@deephaven/utils`.

Reply:
ah, I went ahead and updated